File: test/java/arraysandstrings/LengthOfLongestSubarrayTest.java
package java.arraysandstrings;

import org.junit.jupiter.api.Test;

import static java.arraysandstrings.LengthOfLongestSubarray.findLength;
import static org.junit.jupiter.api.Assertions.assertEquals;

class LengthOfLongestSubarrayTest {

    @Test
    void findLengthTest() {
        assertEquals(2, findLength(new int[]{3, 1, 2, 7, 4, 2, 1, 1, 5}, 8));
    }
}

File: src/test/kotlin/arraysandstrings/LengthOfLongestSubarrayTest.kt
package arraysandstrings

import arraysandstrings.LengthOfLongestSubarray.Companion.findLength
import org.junit.jupiter.api.Assertions.*
import kotlin.test.Test

class LengthOfLongestSubarrayTest {
  @Test
  fun lengthOfLongestSubarrayTest() {
      assertEquals(2, findLength(intArrayOf(3, 1, 2, 7, 4, 2, 1, 1, 5), 8))
  }
}